src/test/java/net/smartcosmos/extension/tenant/converter/tenant/TenantEntityAndUserEntityDtoToCreateTenantResponseConverterTest.java

```diff
@@ -0,0 +1,145 @@
+package net.smartcosmos.extension.tenant.converter.tenant;
+
+import java.util.Collection;
+import java.util.Set;
+import java.util.UUID;
+
+import com.google.common.collect.ImmutableSet;
+
+import org.junit.*;
+
+import net.smartcosmos.cluster.userdetails.domain.RoleEntity;
+import net.smartcosmos.cluster.userdetails.domain.UserEntity;
+import net.smartcosmos.extension.tenant.domain.TenantEntity;
+import net.smartcosmos.extension.tenant.dto.TenantEntityAndUserEntityDto;
+import net.smartcosmos.extension.tenant.dto.tenant.CreateTenantResponse;
+import net.smartcosmos.extension.tenant.dto.user.CreateUserResponse;
+
+import static org.junit.Assert.*;
+
+import static net.smartcosmos.cluster.userdetails.util.UuidUtil.getTenantUrnFromUuid;
+import static net.smartcosmos.cluster.userdetails.util.UuidUtil.getUserUrnFromUuid;
+
+public class TenantEntityAndUserEntityDtoToCreateTenantResponseConverterTest {
+
+    private final TenantEntityAndUserEntityDtoToCreateTenantResponseConverter converter =
+        new TenantEntityAndUserEntityDtoToCreateTenantResponseConverter();
+
+    final Boolean EXPECTED_TENANT_ACTIVE = true;
+    final UUID TENANT_ID = UUID.randomUUID();
+    final String EXPECTED_TENANT_URN = getTenantUrnFromUuid(TENANT_ID);
+    final String EXPECTED_TENANT_NAME = "someTenantName";
+
+    final Boolean EXPECTED_USER_ACTIVE = true;
+    final String EXPECTED_USER_NAME = "someUsername";
+    final UUID USER_ID = UUID.randomUUID();
+    final String EXPECTED_USER_URN = getUserUrnFromUuid(USER_ID);
+    final String EXPECTED_PASSWORD = "somePassword";
+    final String EXPECTED_EMAIL_ADDRESS = "someEmailAddress";
+    final String EXPECTED_GIVEN_NAME = "someGivenName";
+    final String EXPECTED_SURNAME = "someSurname";
+    final String EXPECTED_ROLE_NAME = "Admin";
+    final RoleEntity ROLE = RoleEntity.builder()
+        .name(EXPECTED_ROLE_NAME)
+        .build();
+    final Set<RoleEntity> ROLES = ImmutableSet.of(ROLE);
+
+    TenantEntityAndUserEntityDto ENTITY = TenantEntityAndUserEntityDto.builder()
```

Comment:
`ENTITY` is not really a useful name, `EXPECTED_TENANT` or `EXPECTED_TENANT_ENTITY` is better.